question-service/index.js

```diff
@@ -2,7 +2,9 @@ require('dotenv').config()
 const express = require('express')
 const app = express()
 const mongoose = require('mongoose')
-const port = 5000
+
+// Changed port to 3000, somehow using 5000 doesn't work?
+const port = 3000
```

Comment:
frontend uses port 3000 so maybe use 3001 etc. , hmm strange that port 5000 doesn't work for you

Reply:
Changed port number to 4000. I realised port 5000 doesn't work for me because I have another process on my local machine that also listens on port 5000.

Reply:
The port number should not be a problem, just depends on what you set locally on your machine
Currently I use:
Frontend: 3000
Users: 3001
Questions Ill probably just use 3002